app/models/mentor.rb

```diff
@@ -23,6 +23,8 @@
 #
 
 class Mentor < User
+  default_scope -> { where(type_id: :mentor) }
```

Comment:
__[Rails Best Practices]__

default_scope is evil


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
👍

Reply:
👍